### ISSUE TITLE
✨[Feature] Resize support in dynamic input shape compilation in TRTorch

### ISSUE THREAD

**Issue by p1x31** [NVIDIA/Torch-TensorRT#602]
**Is your feature request related to a problem? Please describe.**

 In order for a neural network to successfully learn to upsample the spatial size of input images is first downsampled with pixel unshuffle. E.g. : 
```python
def pixel_unshuffle(x, scale):
    """ Pixel unshuffle.

    Args:
        x (Tensor): Input feature with shape (b, c, hh, hw).
        scale (int): Downsample ratio.

    Returns:
        Tensor: the pixel unshuffled feature.
    """
    b, c, hh, hw = x.size()
    out_channel = c * (scale**2)
    #assert hh % scale == 0 and hw % scale == 0
    h = int(torch.div(hh , scale, rounding_mode="floor"))
    w = int(torch.div(hw , scale, rounding_mode="floor"))
    x_view = x.view(b, c, h, scale, w, scale)
    return x_view.permute(0, 1, 3, 5, 2, 4).reshape(b, out_channel, h, w)
```
Forward method:
```python
def forward(self, x):
        feat = pixel_unshuffle(x, scale=2)
        feat = self.conv_first(feat)
        body_feat = self.conv_body(self.body(feat))
        feat = feat + body_feat
        # upsample
        feat = self.lrelu(self.conv_up1(F.interpolate(feat, scale_factor=2, mode='nearest')))
        feat = self.lrelu(self.conv_up2(F.interpolate(feat, scale_factor=2, mode='nearest')))
        out = self.conv_last(self.lrelu(self.conv_hr(feat)))
        return out
```

Right now TRTorch throws this [error](https://github.com/NVIDIA/TRTorch/blob/master/core/conversion/converters/impl/shuffle.cpp#L47) when compiling with min_shape, opt_shape, max_shape

**Describe the solution you'd like**

I'd like to have support for `.reshape(b, out_channel, h, w)` operation for dynamic input shapes not just for a single shape
**Describe alternatives you've considered**

**Additional context**

#595

**Comment by ia-davidpichler** [comment]
Can this feature request be reopened?
